.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,43 @@
+name: dependabot
+
+on:
+  pull_request:
+    branches:
+      - dependabot/**
+  push:
+    branches:
+      - dependabot/**
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.16'
```

Comment:
Is it possible to specify a patch version as well?

Reply:
Yes. Selecting 1.16 will just pick latest from that series vs. a specific version. Which way would you like ?